src/main/java/mate/academy/dao/BookDaoImpl.java

```diff
@@ -0,0 +1,164 @@
+package mate.academy.dao;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.Properties;
+import mate.academy.ConnectionUtil;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Book;
+
+@Dao
+public class BookDaoImpl implements BookDao {
+    private static final String PATH_TO_FILE = "src/main/resources/init_db.sql";
+    private Properties dbProp;
+    private String dbUrl;
+
+    public BookDaoImpl(Properties dbProp, String dbUrl) {
+        this.dbProp = dbProp;
+        this.dbUrl = dbUrl;
+    }
+
+    @Override
+    public Book create(Book book) {
+        String query;
+        try {
+            List<String> list = Files.readAllLines(new File(PATH_TO_FILE).toPath());
+            query = String.join(" ", list);
+        } catch (IOException e) {
+            throw new DataProcessingException("Can't read the file " + PATH_TO_FILE, e);
+        }
```

Comment:
```suggestion
        try {
            List<String> list = Files.readAllLines(new File(PATH_TO_FILE).toPath());
            query = String.join(" ", list);
        } catch (IOException e) {
            throw new DataProcessingException("Can't read the file " + PATH_TO_FILE, e);
        }
```

u can move this to the private method, and getting from method only query